Fix and update sea ice plotting example, model agnostic

julia-neme opened the pull request:
Closes #270

schmidt-christina commented on DocumentedExamples/IcePlottingExample.ipynb:
```diff
@@ -4,11 +4,9 @@
    "cell_type": "markdown",
```
<p>I think using mom5 instead of cice and mom6 instead of sis is a bit more intuitive as people might not know how the sea ice model is called, especially if they are oceanographers. Then you can add the sea ice model name in the model_args dictionary.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

anton-seaice replied:
<p>I actually think the opposite ... the mom5 run uses mom5+cice5, the mom6 run uses mom6+sis2 but in the future ACCESS-OM3 will use mom6+cice6. So labelling by the sea-ice model is clearer. In the future we will have mom6 results where these variable names will need to come from cice6, not sis.</p>

schmidt-christina commented on DocumentedExamples/IcePlottingExample.ipynb:
```diff
@@ -4,11 +4,9 @@
    "cell_type": "markdown",
```
<pre class="ql-syntax" spellcheck="false">Line #7.                  "sis": {"expt": "panant-01-zstar-v13",
</pre><p>I would suggest to use the current version which is panat-01-zstar-ACESSyr2 and then you woueld need to change the time period</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

navidcy commented:
@julia-neme could you address these comments?

julia-neme commented:
Well, I don't know what happened but I addressed these comments and pushed to my [fork](https://github.com/julia-neme/cosima-recipes/blob/master/DocumentedExamples/IcePlottingExample.ipynb) but now the IcePlottingExample.ipynb is a completelly different notebook. Not a clue what happened

navidcy commented:
Might be me merging #280  yesterday?

navidcy commented:
ok, perhaps because of bad organization on my part during the hackathon both @julia-neme and @anton-seaice worked on the same notebook! and now one has overwritten the others changes. could you both have a look at the notebook in main and confirm if everything is OK?

(Sorry..)

anton-seaice commented:
@julia-neme 's plots are nicer than mine, so I could merge her plots/mom6 result into the version in master if desired?

julia-neme commented:
I don't mind, whatever you think it's best Navid. I thought Anton was working at SeaIceSeasonality notebook, sorry!

anton-seaice commented:
I raised a new PR #294 to tidy up the merge :)
